src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -177,26 +176,29 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
         this.fetchBackupInfo();
     }
 
-    private async fetchBackupInfo(): Promise<{ backupInfo?: IKeyBackupInfo; backupSigStatus?: TrustInfo }> {
+    private async fetchBackupInfo(): Promise<{ backupInfo?: IKeyBackupInfo; backupEnabled?: boolean }> {
```

Comment:
in the result of this, how does `backupEnabled === undefined` differ from `backupEnabled === false`? Document it please.

Reply:
Will update, it's was only for legacy as there was no await on the backup (but just `getKeyBackupEnabled` that was undefined until a check to backup has been done)